Directory.Build.targets

```diff
@@ -46,6 +46,14 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="_GetNativeAssetsForPackage">
+    <ItemGroup>
+      <TfmSpecificPackageFile Include="$(BaseOutputPath)../Native/Release/*.so">
```

Comment:
double checking - if you build this on 64 bit linux, this will work correctly with arm32?

Reply:
Yes, we technically build it using a crossrootfs that is targeting arm, which is what makes it work. This is the same way we do it for corefx and coreclr 😄

Reply:
I'm a bit surprised that nothing in the source path is unique for architecture, yet you're putting everything under "linux-arm" RID in the package.

Reply:
Right now, we only have one native build which is linux arm. We only plan to have one more specific RID native binary (arm64) so I had planned that this would have to get fixed once that is added.

Reply:
I see.  Consider a comment here that explains what's going on and how you intend for it to change.  Both @krwq and myself were confused by it.

Reply:
Will do.

Reply:
Should you add validation for official builds and CI builds thjat this itemgroup is not empty and error if it is? This to guard your builds and make sure your package always contains this native assets.